app/connectors/PPConnector.scala

```diff
@@ -38,7 +38,7 @@ import scala.concurrent.ExecutionContext.Implicits.global
 @ImplementedBy(classOf[PrePopConnector])
 trait PPConnector {
 
-  val companyRegUrl: String
+//  val companyRegUrl: String
```

Comment:
I had to comment or remove these lines, as I wanted to make use of:
  val companyRegUrl = baseUrl("company-registration")
in CompanyRegistrationConnector

will explain tomorrow